reference/5.1/Microsoft.PowerShell.Core/About/about_Character_Encoding.md

```diff
@@ -0,0 +1,230 @@
+---
+title: about_Character_Encoding
+description: Describes how PowerShell uses character encoding for input and output of string data.
+ms.date: 10/21/2020
+Locale: en-US
+online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_character_properties?view=powershell-5.1&WT.mc_id=ps-gethelp
+schema: 2.0.0
+---
+# about_Character_Encoding
+
+## Short description
+Describes how PowerShell uses character encoding for input and output of string
+data.
+
+## Long description
+
+Unicode is a worldwide character-encoding standard. The system uses Unicode
+exclusively for character and string manipulation. For a detailed description
+of all aspects of Unicode, refer to
+[The Unicode Standard](https://www.unicode.org/standard/standard.html).
+
+Windows supports Unicode and traditional character sets. Traditional character
+sets, such as Windows code pages, use 8-bit code values or combinations of
+8-bit values to represent the characters used in a specific language or
+geographical region.
+
+PowerShell uses a Unicode character set by default. However, several cmdlets
+have an **Encoding** parameter that changes the encoding to a different
+character set. This parameter allows you to choose the specific the character
+encoding you need for interoperability with other systems and applications.
+
+The following cmdlets have the **Encoding** parameter:
+
+- Microsoft.PowerShell.Management
+  - Add-Content
+  - Get-Content
+  - Set-Content
+- Microsoft.PowerShell.Utility
+  - Export-Clixml
+  - Export-Csv
+  - Export-PSSession
+  - Format-Hex
+  - Import-Csv
+  - Out-File
+  - Select-String
+  - Send-MailMessage
+
+## The byte-order-mark
+
+The byte-order-mark (BOM) is a _Unicode signature_ in the first few bytes of a
+file or text stream that indicate which Unicode encoding is being used. For
+more information, see the
+[Byte order mark](https://en.wikipedia.org/wiki/Byte_order_mark) article in
+Wikipedia.
+
+In Windows PowerShell, any Unicode encoding, except `Utf7`, always creates a
+BOM. PowerShell Core defaults to `utf8NoBOM` for all text output.
+
+For best overall compatibility, avoid using BOMs in UTF-8 files. Unix platforms
+and Unix-heritage utilities also used on Windows Platforms don't support BOMs.
+
+Similarly, `utf7` encoding should be avoided. UTF-7 is not a standard Unicode
+encoding and is written without a BOM in all versions of PowerShell.
+
+Creating PowerShell scripts on a Unix-like platform or using a cross-platform
+editor on Windows, such as Visual Studio Code, results in a file encoded using
+`UTF8NoBOM`. These files work fine on PowerShell Core, but may break on Windows
+PowerShell if the file contains non-Ascii characters.
+
+If you do need to use non-Ascii characters in your scripts, save them as UTF-8
```

Comment:
Remove "do", feels filler.

Reply:
Should probably add what the UTF-8 with BOM value is since you've referenced the `UTF8NoBOM` already, otherwise they'll have to look it up.

Reply:
Fixed